FORMAT.md

```diff
@@ -135,6 +155,13 @@ The rigid body refers to a solid body with mass and one or more outputs. Default
 - `output_trans` (translation vector, m): The default position the output frames. Defaults to (0,0,0).
 - `ixx`, `iyy`, `izz`, `ixy`, `ixz`, `iyz` (floating point formulae, kg m^2): The 6 elements of the inertia tensor, relative to the COM frame as given above.  Each defaults to 0 (note, this means overall default is a point mass).
 
+**Content:**
+In this order:
+- Zero or more of the following:
+  - `<mesh>`: describes visual geometry of the rigid body; see the `<mesh>` element section below.
+- Zero or more of the following:
+  - `<output>` Used to describe tree-like kinematic structures. See the `<output>` element section below.
```

Comment:
After talking with @cwbollinger , one of the concerns he had wanted the grouping for was just ease of parsing.  Since this isn't an issue in the C++ XML parser we are using, this isn't as significant.

I'm still open to either nesting in <meshes> and <outputs> or not here if there are strong feelings from @ennerf .